docs/content/about/tech-specs.md

```diff
@@ -16,63 +16,28 @@ The following tables list the software versions NGINX Service Mesh supports and
 ### Supported Versions
 
 {{% table %}}
-| NGINX Service Mesh | Kubernetes  | NGINX Plus Ingress Controller (ingress) | NGINX Plus Ingress Controller (egress) | OpenShift  | Helm     | Rancher  |
-|--------------------|-------------|-----------------------------------------|----------------------------------------|------------|----------|----------|
-| v1.7.0             | 1.22 - 1.24 | 1.12, 2.2, 2.3                          | 2.2, 2.3                               | 4.9 - 4.10 | 3.2+     | 2.6, 2.7 |
-| v1.6.0             | 1.22 - 1.24 | 1.12, 2.2, 2.3                          | 2.2, 2.3                               | 4.9 - 4.10 | 3.2+     | 2.5, 2.6 |
-| v1.5.0             | 1.22 - 1.24 | 1.12, 2.1, 2.2                          | 2.2                                    | 4.9 - 4.10 | 3.2+     | 2.5, 2.6 |
-| v1.4.1             | 1.18 - 1.22 | 1.12, 2.1, 2.2                          | 2.2                                    | 4.8        | 3.2+     | 2.5, 2.6 |
-| v1.4.0             | 1.18 - 1.22 | 1.12, 2.0, 2.1                          | 1.12, 2.0, 2.1                         | 4.8        | 3.2+     | 2.5, 2.6 |  
-| v1.3.1             | 1.18 - 1.22 | 1.12, 2.0                               | 1.12, 2.0                              | 4.8        | 3.2+     | 2.5, 2.6 |     
-| v1.3.0             | 1.18 - 1.22 | 1.12, 2.0                               | 1.12, 2.0                              | 4.8        | 3.2+     | 2.5, 2.6 |     
-| v1.2.1             | 1.16 - 1.21 | 1.12.0                                  | 1.12.0                                 | 4.8        | 3.5, 3.6 | 2.6      |
-| v1.2.0             | 1.16 - 1.21 | 1.12.0                                  | 1.12.0                                 | 4.8        | 3.5, 3.6 | 2.6      |
-| v1.1.0             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-| v1.0.1             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-| v1.0.0             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-
+| NGINX Service Mesh | Kubernetes | OpenShift | NGINX Ingress Controller | NGINX Plus Ingress Controller | Helm     | Rancher  |
```

Comment:
I don't understand:
`NGINX Ingress Controller | NGINX Plus Ingress Controller`
It is the same codebase it should be the same version.  Should it not?

Reply:
We had to make some changes in order for OSS NIC to work. Those changes went in for NIC v3. N+IC will work for v2 and v3. If you want to drop support for v2, then we can align on a single version.

Reply:
The two flavors is very confusing to customers and even internal folks that are not on the team.
Lets list our NIC project once.  List both.

And have a note someone where we call out the nuance regarding the free flavor of NIC.
Definitely in the release notes.  And I assume the install docs will focus on NIC v3.
Even a temporary note under the table in this document can reinforce until NSM drops NIC 2.x.